Composer/packages/client/package.json

```diff
@@ -20,7 +20,7 @@
     "@bfc/extensions": "*",
     "@bfc/indexers": "*",
     "@bfc/shared": "*",
-    "@bfcomposer/bf-lu": "^1.2.9",
+    "@microsoft/bf-lu": "4.8.0-preview.110925",
```

Comment:
Is it possible not to make it a fixed version? Automatically use latest version.